inyoka_theme_default/static/style/inyoka/planet.less

```diff
@@ -10,16 +10,10 @@
 
 .planet {
   &-newestarticles {
-    .list-inline {
-      text-align:justifed;
-
-      li {
-        padding: 0;
-
-        &:not(:last-child):after {
-          color: @gray-light;
-          content: ' |';
```

Comment:
but if I see it correct that's only a display:inline-block. and i prefer the " | " as a speration of the links.